### DISCUSSION TITLE
How to pin vcpkg version?

### DISCUSSION THREAD

**Discussion by Seng-Jik** [microsoft/vcpkg#27809] [Q&A]
In my project CI, I just download vcpkg by git and bootstrap it, but it sometimes fail, how can I pin vcpkg to a stable version?

Now problem:
* Recently, github CI Runner (linux-latest) can not bootstrap vcpkg with `write_contents("/home/runner/.vcpkg/vcpkg.path.txt"): No such file or directory`
* I can not compile ffmpeg for triplet arm-windows-static
* These problems sometimes happen, and sometimes everything works well.

**Comment by JackBoosY** [comment]
Just don't `git pull`?
